test/hotspot/jtreg/runtime/valhalla/inlinetypes/MultiANewArrayTest/MultiANewArrayTest.java

```diff
@@ -49,8 +49,7 @@ public static void main(String[] args) {
         } catch(Error e) {
             ex = e;
         }
-        Asserts.assertNotNull(ex, "An ICCE should have been thrown");
-        Asserts.assertEquals(ex.getClass(), IncompatibleClassChangeError.class, "Error is not an ICCE");
+        Asserts.assertNull(ex, "No ICCE should have been thrown");
```

Comment:
The negation of "An ICCE should have been thrown" is "No error should have been thrown",
talking about ICCE it that context can be misleading.

Reply:
Error message updated in the last commit.

Thanks,

Fred